tsconfig.json

```diff
@@ -22,5 +23,8 @@
     "dist/**/*",
     "node_modules/**"
   ],
-  "extends": "@tsconfig/node14/tsconfig.json"
+  "extends": "@tsconfig/node14/tsconfig.json",
+  "paths": {
```

Comment:
This configuration option allows us to specify backfill TypeScript types for modules which lack them. The way this option works is explained here: https://github.com/MetaMask/metamask-module-template/pull/95#issuecomment-1079484410